test/parallel/test-primordials-restricted-access.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+// Flags: --no-warnings --expose-internals
+
+const common = require('../common');
+
+if (Date.now() < 999 && Date.now() > 0) {
```

Comment:
Why is this and the `http.utcDate()` check necessary? Shouldn't the `common.mustNotCall()` be enough?